devices/Hdc1080/Hdc1080.cs

```diff
@@ -0,0 +1,173 @@
+            SetHumidityResolution(humidityResolution);
+        }
+
+        /// <summary>
+        /// Reads temperature, check data sheet, page 14, 8.6.1 section
+        /// </summary>
+        /// <returns>
+        /// Temperature
+        /// </returns>
+        [Telemetry("Temperature")]
+        public Temperature ReadTemperature() => Temperature.FromDegreesCelsius((ReadData(RegisterAddress.TemperatureMeasurement) * 165 / 65536f) - 40);
+
+        /// <summary>
+        /// Reads relative humidity, check data sheet, page 14, 8.6.2 section
+        /// </summary>
+        /// <returns>
+        /// Relative humidity
+        /// </returns>
+        [Telemetry("Humidity")]
+        public RelativeHumidity ReadHumidity() => RelativeHumidity.FromPercent(ReadData(RegisterAddress.HumidityMeasurement) * 100 / 65536);
+
+        /// <summary>
+        /// Reads device id, check data sheet, page 14, 8.6.6 section
+        /// </summary>
+        /// <returns>
+        /// Device id
+        /// </returns>
+        public string ReadDeviceId() => ReadData(RegisterAddress.DeviceId).ToString();
+
+        /// <summary>
+        /// Reads manufacturer id, check data sheet, page 14, 8.6.5 section
+        /// </summary>
+        /// <returns>
+        /// Manufacturer id
+        /// </returns>
+        public string ReadManufacturerId() => ReadData(RegisterAddress.ManufacturerId).ToString();
+
+        /// <summary>
+        /// Reads serial number, check data sheet, page 14, 8.6.4 section
+        /// </summary>
+        /// <returns>
+        /// Serial number
+        /// </returns>
+        public string ReadSerialNumber() => $"{ReadData(RegisterAddress.SerialIdFirstByte)}{ReadData(RegisterAddress.SerialIdSecondByte)}{ReadData(RegisterAddress.SerialIdThirdByte)}";
+
+        /// <summary>
+        /// Sets humidity resolution
+        /// </summary>
+        /// <param name="resolution">Humidity resolution</param>
+        public void SetHumidityResolution(HumidityResolution resolution)
+        {
+            ConfigurationRegister register = (ConfigurationRegister)ReadRegister();
+            register.HumidityMeasurementResolution = (byte)resolution;
+
+            WriteRegister(register);
+        }
+
+        /// <summary>
+        /// Sets temperature resolution
+        /// </summary>
+        /// <param name="resolution">Temperature resolution</param>
+        public void SetTemperatureResolution(TemperatureResolution resolution)
+        {
+            ConfigurationRegister register = (ConfigurationRegister)ReadRegister();
+            register.TemperatureMeasurementResolution = (byte)resolution;
+
+            WriteRegister(register);
+        }
+
+        /// <summary>
+        /// Runs heater for given number of milliseconds, check data sheet, page 9, 8.3.3 section
+        /// </summary>
+        /// <param name="milliseconds">Number of milliseconds</param>
+        public void HeatUp(int milliseconds)
```

Comment:
It would be better to have a TimeSpan instead of the milliseconds.

Reply:
Fixed